include/boost/astronomy/coordinate/io.hpp

```diff
@@ -22,9 +22,13 @@ template
     class YQuantity,
     class ZQuantity
 >
-std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
+std::ostream& operator<< (std::ostream &out, cartesian_representation
+	<CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( " 
+    	    << point.get_x() << " , " 
+            << point.get_y() << " , " 
```

Comment:
Due to some reason, indentation between line `29` and `30` appears different in this preview. But that's not the case in the original file. So not a problem.